gwi-project/src/Utils/stringUtils.ts

```diff
@@ -0,0 +1,6 @@
+export function capitalizeFirstLetter(string: string) {
```

Comment:
Why do you have both function declarations as `export const` and `export function` ? it breaks the consistency of the code , do you agree?

Reply:
When I am using a utility function I also like using the name function but I agree with you!